packages/data-provider/src/core/provider.ts

```diff
@@ -22,11 +26,14 @@ const provider = (configData: AppConfig) => {
 		configs: deps,
 		routers: {
 			dictionary: dictionaryRouters(deps),
-			submission: submissionRouters(deps),
+			submission: submissionRouters(_.omit(deps, ['config'])),
+			submittedData: submittedDataRouters(_.omit(deps, ['config'])),
 		},
 		utils: {
 			dictionary: getDictionaryUtils(deps),
-			category: getCategoryUtils(deps),
+			category: getCategoryUtils(_.omit(deps, ['config'])),
+			submission: getSubmissionUtils(_.omit(deps, ['config'])),
+			submittedData: getSubmittedDataUtils(_.omit(deps, ['config'])),
```

Comment:
looks like a few `omit(deps, ['config'])`... are we expecting to have each of these lines omit different things later on? or could these all be abstracted to a single DRY definition like 
```js
const blah = omit...;

...
routers: { submission: submissionRouters(blah) ... }
```